pulp_puppet_plugins/pulp_puppet/plugins/migrations/0005_puppet_module_name_change.py

```diff
@@ -0,0 +1,59 @@
+from gettext import gettext as _
+import logging
+
+from mongoengine import NotUniqueError, Q
+from pulp.server.controllers import repository as repo_controller
+from pulp.server.db import model
+
+from pulp_puppet.plugins.db.models import Module
+
+_log = logging.getLogger('pulp')
+
+
+def migrate(*args, **kwargs):
+    """
+    For each puppet module check and if needed update module name format.
+
+    There was a discrepancy in the way puppet module's name was stored in pulp, depending
+    if it was synced from filesystem or uploaded. This migration finds puppet module units
+    that have wrong format name and replaces it with a correct format name.
+    """
+
+    modules = Module.objects(Q(name__contains='/') | Q(name__contains='-'))
```

Comment:
For forward compatability as we switch to Django, don't use the callable nature of `objects`. In other words, use a method provided by `objects` but not `objects` directly. In this case you're filtering so change this to:

`Module.objects.filter(Q(name__contains='/') | Q(name__contains='-'))`